tests/cluster/tests/29-slot-migration-replication.tcl

```diff
@@ -0,0 +1,351 @@
+#Tests that validate slot migration states are replicated propoerly
+
+source "../tests/includes/init-tests.tcl"
+source "../tests/includes/utils.tcl"
+
+variable master_count 3
+variable replica_count 1
+
+proc abort {message} {
+    puts $message
+    #gets stdin t
+    throw 1 $message
+}
+
+proc get_node_descr {id} {
```

Comment:
Maybe use "get_cluster_nodes" so that it's formatted into a dict instead of an array.